tests/test_single_topology.py

```diff
@@ -791,7 +791,7 @@ def test_combine_with_host():
         potentials.PeriodicTorsion,
         potentials.NonbondedPairListPrecomputed,
         potentials.Nonbonded,
-        potentials.SummedPotential,  # P-L + L-W interactions
+        potentials.NonbondedInteractionGroup,  # P-L + L-W interactions
```

Comment:
nit: to be consistent that it is computing the ligand energies with respect to protein and water (IE which are the row atoms)
```suggestion
        potentials.NonbondedInteractionGroup,  # L-P + L-W interactions
```